en/bundle.ftl

```diff
@@ -11,20 +11,22 @@ vpn-relay-go-vpn-cta = Download { -brand-name-mozilla-vpn }
 
 ## VPN and Relay Bundle What's New Announcement
 
-whatsnew-feature-bundle-header = Introducing: { -brand-name-relay } + { -brand-name-vpn } subscription plan
-whatsnew-feature-bundle-snippet = Upgrade your subscription to get both { -brand-name-firefox-relay-premium } + { -brand-name-mozilla-vpn }…
+# Variables:
+#   $savings (string) - the percentage saved (including % symbol) for a given plan. Examples: 50%, 70%
+whatsnew-feature-bundle-header = Add { -brand-name-mozilla-vpn } for { $savings } off!
+whatsnew-feature-bundle-snippet = Get a year of { -brand-name-relay-premium } Phone Masking & Email Masking plus { -brand-name-mozilla-vpn }…
```

Comment:
Updates the what's new bundle entry 
<img width="396" alt="image" src="https://user-images.githubusercontent.com/13066134/197017934-58629647-f40e-4773-af28-ef51e48679eb.png">